ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4375,4 +4375,65 @@ EC2SC:
   - supply_air_temperature_sensor
   - supply_air_temperature_setpoint
   implements:
-  - CONTROL
\ No newline at end of file
+  - CONTROL
+
+CHWISVPCM:
+  description: "Chllled water isolation valve percentage control and monitoring"
+  is_abstract: true
+  implements:
+  - OPERATIONAL
+  opt_uses:
+  - chilled_water_flowrate_alarm
+  - chilled_water_isolation_valve_mode
+  - chilled_water_isolation_valve_percentage_setpoint
```

Comment:
Yes,  it is different from that type because the chilled_water_isolation_valve_percentage_setpoint is one where user input while the chilled_water_isolation_valve_percentage_command is the feedback on the user input.

Reply:
the sensor is the feedback, a command is not a feedback -- its a command. please change.

Reply:
the setpoint has been removed